### DISCUSSION TITLE
Name of the are Listed below

### DISCUSSION THREAD

**Discussion by saltsoftdrink** [catppuccin/catppuccin#2214] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

These are some most common tools/websites users use all over the world

I've added bullet for more IMP sites to work on

* Twitter
codepen.io
* Dev.to
* Deviantart
* Leetcode
* Amazon
* Flipkart

### Submission Guidelines

- [x] I have read and followed Catppuccin's submission guidelines.

**Comment by justTOBBI** [comment]
Signal can't be themed at the moment if I got the last conversation correct
And Ventoy can be themed if you head over to https://github.com/catppuccin/grub 
:)

**Comment by saltsoftdrink** [comment]
> Signal can't be themed at the moment if I got the last conversation correct And Ventoy can be themed if you head over to https://github.com/catppuccin/grub :)

Thanks Buddy for Ventoy

**Comment by sgoudham** [comment]
Hi there :wave:, please keep the port requests limited to a **single application/website/tool**. The value of separate discussions is lost by aggregating multiple requests into one discussion.

As your request seems to mostly consist of websites, make sure to check out https://github.com/catppuccin/userstyles for a list of our existing themes for websites before raising further ones.

For what its worth, I'd personally encourage you to only raise port requests for applications and tools that you yourself are passionate about instead of just the "most popular sites in the world." Of course, it's nice having Catppuccin for all the above ports but I find that more people are willing to maintain the applications/websites/tools when people genuinely use it as part of their daily workflow. (E.g. do we really need Catppuccin for speedtest.com? :P)

Thanks.

**Comment by saltsoftdrink** [comment]
> Hi there 👋, please keep the port requests limited to a **single application/website/tool**. The value of separate discussions is lost by aggregating multiple requests into one discussion.
> 
> As your request seems to mostly consist of websites, make sure to check out https://github.com/catppuccin/userstyles for a list of our existing themes for websites before raising further ones.
> 
> For what its worth, I'd personally encourage you to only raise port requests for applications and tools that you yourself are passionate about instead of the just the "most popular sites in the world." Of course, it's nice having Catppuccin for all the above ports but I find that more people are willing to maintain the applications/websites/tools when people genuinely use it as part of their daily workflow. (E.g. do we really need Catppuccin for speedtest.com? :P)
> 
> Thanks.

Thanks man Sorry for the inconvenience, ensuring a more focused & efficient approach moving forward
